docs/upgrading-pega-deployment-zero-downtime.md

```diff
@@ -0,0 +1,179 @@
+   | installer.targetRulesSchema   | For upgrades from 8.4.2 and later, specify a new rules schema name that the upgrade process creates in your existing database to support the upgrade process within the quotes.  For upgrades starting as earlier versions, you can leave this value empty, as is (do not leave it blank).| targetRulesSchema: ""  |
+   | installer.targetDataSchema   | For upgrades from 8.4.2 and later, specify the new target data schema name that you created in your existing database to support the upgrade process within the quotes and you delete from your database after you complete your upgrade. For upgrades starting as earlier versions, you can leave this value empty, as is (do not leave it blank). | targetDataSchema: "" |
+
+2. Save the file.
+
+### Upgrading your Pega Platform deployment using the command line
+
+In this document, you specify that the Helm chart always “deploys” by using the setting, actions.execute: "upgrade-deploy argument". After you have your customizations saved in your pega Helm chart, you are ready to upgrade.
+
+1. Do one of the following:
+
+   - Open Windows PowerShell running as Administrator on your local system and change the location to the top folder of your `\<platform\>-demo` folder that you created in [Preparing your local Windows 10 system](https://github.com/pegasystems/pega-helm-charts/blob/master/docs/prepping-local-system-runbook-windows.md).
+
+      `$ cd <local filepath>\<platform>-demo`
+
+   - Open a Linux bash shell and change the location to the top folder of your `\<platform\>-demo` directory that you created in [Preparing your local Linux system](https://github.com/pegasystems/pega-helm-charts/blob/master/docs/prepping-local-system-runbook-linux.md).
+
+      `$ cd /home/<local filepath>/<platform>-demo`
+
+2. Upgrade Pega Platform using your updated `pega` Helm chart.
+
+   ```yaml
+   helm upgrade mypega-<platform>-demo pega/pega --namespace mypega-<platform>-demo --values pega.yaml
+   ```
+
+   A successful upgrade immediately returns details that shows progress for your `mypega-<platform>-demo` deployment.
+
+3. Refresh the Kubernetes dashboard that you opened in the previous section. If you closed the dashboard, start the proxy server for the Kubernetes dashboard and then relaunch the web browser.
+
+4. In the dashboard, in **Namespace** select the `mypega-<platform>-demo` view and then click on the **Pods** view. Initially, you can some pods have a red status, which means they are initializing:
+
+    You can follow the progress of your upgrade using the dashboard. Initially, while the resources make requests to complete the configuration, you will see red warnings while the configuration is finishing, which is expected behavior.
+
+5. To view the status of an installation, on the Kubernetes dashboard, select **Jobs**, locate the **pega-db-install** job, and click the logs icon on the right side of that row.
```

Comment:
zerodowntime upgrade job name is  - pega-zdt-upgrade 
Please refer job names - 

https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/charts/installer/templates/pega-installer-job.yaml

And you can look for the defined variable name in the repo.